lib/reader.js

```diff
@@ -97,15 +97,8 @@ function Reader (props, currentStat) {
 
   self._path = self.path = path.resolve(props.path)
   if (process.platform === 'win32') {
```

Comment:
Node has been doing the long-path transformation internally since version [0.5.something](https://github.com/nodejs/node/commit/1f16a7b6e541694067178a022b9f7a082a6ce7f1). Unless you guys still support node v0.4, you should indeed nuke this bit.